Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM bitnami/python:3.7-prod
```

Comment:
Are you going to move to the python slim image here too or stick with bitnami for now?

Reply:
Yes, this entire thing needs to be updated to the python slim image. I was away yesterday so didn't have the time to work on the bot or seasonalbot dockerfiles.

Reply:
This is now actioned.